.github/workflows/continuous-integration-workflow.yml

```diff
@@ -21,4 +21,40 @@ jobs:
       run: docker run -e MONGODB_CONNECTIONSTRING=FAKE_CONNECTIONSTRING -e MONGODB_DATABASE=doMeDatabase -i do-me:test todo_app/tests
 
     - name: Launch Docker container do-me:test and run e2e tests
-      run: docker run -e MONGODB_CONNECTIONSTRING=${{ secrets.MONGODB_CONNECTIONSTRING }} -e MONGODB_DATABASE=doMeDatabase -i do-me:test todo_app/tests_e2e
\ No newline at end of file
+      run: docker run -e MONGODB_CONNECTIONSTRING=${{ secrets.MONGODB_CONNECTIONSTRING }} -e MONGODB_DATABASE=doMeDatabase -i do-me:test todo_app/tests_e2e
+    
+  deploy:
+    name: Deploy to Docker Hub
+    if: ${{ github.ref == 'refs/heads/exercise-8' && github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    needs: build
+    steps:   
+    - 
+        uses: actions/checkout@v2     
+    -
+        name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - 
+        name: Build Docker Image
+        run: docker build --target production --tag jackiemaw/do-me:latest .
+    - 
+        name: Push Docker Image to DockerHub
+        run: docker push jackiemaw/do-me:latest
```

Comment:
Super minor suggestion, but you could pull some things out like `jackiemaw/do-me:latest` into a variable so you don't have to worry about keeping things in sync in case you rename it

Reply:
done! better